nevergrad/benchmark/experiments.py

```diff
@@ -191,6 +228,24 @@ def illcondipara(seed: Optional[int] = None) -> Iterator[Experiment]:
                 yield Experiment(function.duplicate(), optim, budget=budget, num_workers=1, seed=next(seedg))
 
 
+@registry.register
+def constrainedillcondipara(seed: Optional[int] = None) -> Iterator[Experiment]:
```

Comment:
constrained_ill_conditioned_params

Reply:
Done (with a slight variant - it's parallel instead of params, which validates your claim that I should be explicit and use long names :-)